migrations/genesis.config.js

```diff
@@ -4,22 +4,21 @@ const TOKEN_UNIT = 10 ** 18
 
 module.exports = {
     initialSupply: new BigNumber(10000000).mul(TOKEN_UNIT),
-    // This should always be 0. This field is an artifact of an older
-    // design where crowd supply distribution was done during genesis.
-    // However, now crowd supply distribution is done post-genesis, so
-    // we just keep this field at 0
-    crowdSupply: new BigNumber(0),
-    companySupply: new BigNumber(7500000).mul(TOKEN_UNIT),
+    crowdSupply: new BigNumber(180000).mul(TOKEN_UNIT),
+    companySupply: new BigNumber(7320000).mul(TOKEN_UNIT),
     teamSupply: new BigNumber(1100000).mul(TOKEN_UNIT),
     investorsSupply: new BigNumber(900000).mul(TOKEN_UNIT),
     communitySupply: new BigNumber(500000).mul(TOKEN_UNIT),
-    // This value will only be used when deploying to a non-production network
-    // i.e. Any network besides mainnet and a test network being used for a dry run
-    faucetSupply: new BigNumber(180000).mul(TOKEN_UNIT),
     bankMultisig: "0x0161e041aad467a890839d5b08b138c1e6373072", // Should replace this placeholder address in a real deployment
     governanceMultisig: "0x0161e041aad467a890839d5b08b138c1e6373072", // Should replace this placeholder address in a real deployment
-    dummyTokenDistribution: {
-        timeToEnd: new BigNumber(60).times(60).times(24).times(7) // 1 week in seconds
+    timeToGrantsStart: new BigNumber(60).times(60).times(24).times(7), // 1 week in seconds
```

Comment:
For dry run maybe we set this to be just a couple hours past the expected end of the run.